### DISCUSSION TITLE
Bookmarks file name and location.

### DISCUSSION THREAD

**Discussion by Userbygithub** [Eloston/ungoogled-chromium#1613] [Q&A]
I installed Ungoogled chromium, and then copied bookmarks from Chrome into it.  I know the file itself is bookmarks, and where it is located in chrome and in portable versions.  I do bookmarks three ways, copy and paste from bookmarks manager,  import from html file, or copy Bookmarks file from user folder. However, after copying the URLs,  I looked through all the folders of  Ungoogled Chromium and I can't even find the bookmarks file that should be there. It should be over 13 megs, and yet, even doing a search on the whole drive, there is no new file that pops up as Bookmarks that is 13 megs.  So I was wondering, is it a different file name. I guess I could look for file size only??

**Comment by oregonpillow** [comment]
For me the file is `/Users/$USER/Library/Application Support/Chromium/Default/Bookmarks` , where Bookmarks (extension-less) is a file not a folder.
